ionoscloud/resource_server_test.go

```diff
@@ -1088,7 +1083,6 @@ resource ` + constant.ServerResource + ` ` + constant.ServerTestResource + ` {
   cores = 1
   ram = 1024
   availability_zone = "ZONE_1"
-  cpu_family = "INTEL_XEON"
```

Comment:
@cristiGuranIonos Is this `cpu_family` not available anymore? Why are we removing it?

Reply:
keeps the tests easier to maintain. Either a cpu family is removed, it's not available or changes. This allows the server to choose it's own cpu family.